src/shared/components/checkbox/Checkbox.scss

```diff
@@ -38,16 +34,31 @@
   }
 }
 
-.defaultLabel {
-  color: $black;
+.labelDefault {
+  color: var(--checkbox-label-color, #{$black});
+  font-size: var(--checkbox-label-font-size, 13px);
+  font-weight: var(--checkbox-label-font-weight, #{$normal});
   line-height: 30px;
-  font-size: 12px;
   padding-left: 8px;
   padding-right: 2px;
   max-width: calc(100% - 17px);
   word-wrap: break-word;
 }
 
+.themeLighter {
+  --checkbox-label-font-size: 12px;
```

Comment:
I have a suspicion that this might make the variable harder to override from a parent component. What if instead of defining the variable here, we instead write `font-size: var(--checkbox-label-font-size, 13px);`? Then the parent can still change the `--checkbox-label-font-size` variable if it needs to.

Same for other properties.

Reply:
So if we do this here, won't we need to do this in the other two theme classes as well?

Reply:
✅